### PR TITLE
Extend createsuperuser command to allow SuperCreate assignment.

### LINKED ISSUE #2593: Provision docker with a super creator
When starting a new Docker instance of AMO, the automated tests need a way to generate a [super creator](https://addons-server.readthedocs.io/en/latest/topics/api/accounts.html#super-creation), I.E. a user with credentials to make POSTs to `/api/v3/accounts/super-create/`.

One possible solution is to introduce a management command that can be called like:

```
python manage.py newsupercreator
```

The script would:
- create a new user
- assign it to the `Accounts:SuperCreate` group
- tick the 'yes I accept' developer agreement box
- generate API credentials
- output credentials in JSON

### PR DESCRIPTION
**Pull request by EnTeQuAk**
This re-adds --no-input arguments and handles it properly.

Fixes #2593

### CODE REVIEW

File: src/olympia/users/management/commands/createsuperuser.py [hunk 1]
```diff
@@ -4,29 +4,81 @@
...
 and email address and that's it.
 '''
-    required_fields = ['username', 'email']
+    # TODO: Use `UserProfile.REQUIRED_FIELDS`? Not sure why `username`
+    # isn't in there...
+    required_fields = ('username', 'email')
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--allow-supercreate',
```
Comment (EnTeQuAk) [inline src/olympia/users/management/commands/createsuperuser.py]: granted, I just felt like it'd be too long. On the other hand I could have just added the user to the group automatically but somehow this didn't feel right.

Comment (diox) [inline src/olympia/users/management/commands/createsuperuser.py]: Any of these alternatives is better than the current situation, IMHO. `manage.py createsuperuser --allow-supercreate` is super confusing.

Comment (EnTeQuAk) [inline src/olympia/users/management/commands/createsuperuser.py]: How about ``--is-supercreator``? Similar to how PostgreSQL tools or UNIX `createuser` ask for advanced permissions on user creation.

Comment (eviljeff) [inline src/olympia/users/management/commands/createsuperuser.py]: I'd go with `--add-to-supercreate-group` as @diox said or have a separate command to `createsuperuser` (`createsuperusercreator`? 😆 ) that does it automatically.

Comment (EnTeQuAk) [inline src/olympia/users/management/commands/createsuperuser.py]: Fixed in 839321c56f :)

Comment (eviljeff) [inline src/olympia/users/management/commands/createsuperuser.py]: I think you need to update your tests

**Comment by EnTeQuAk** [comment]
Tests are fixed and green again, re-r?

**Pull request merged**